examples/aggregate.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace MongoDB\Examples;
+
+use MongoDB\Client;
+
+use function assert;
+use function dirname;
+use function getenv;
+use function is_object;
+use function MongoDB\BSON\fromPHP;
+use function MongoDB\BSON\toRelaxedExtendedJSON;
+use function printf;
+use function rand;
+
+require dirname(__FILE__) . '/../vendor/autoload.php';
+
+function toJSON(object $document): string
+{
+    return toRelaxedExtendedJSON(fromPHP($document));
+}
+
+$client = new Client(getenv('MONGODB_URI') ?: 'mongodb://127.0.0.1/');
+
+$collection = $client->test->coll;
+$collection->drop();
+
+$documents = [];
+
+for ($i = 0; $i < 100; $i++) {
+    $documents[] = ['randomValue' => rand(0, 1000)];
+}
+
+$collection->insertMany($documents);
+
+$pipeline = [
+    [
+        '$group' => [
+            '_id' => null,
+            'totalCount' => ['$sum' => 1],
+            'evenCount' => [
+                '$sum' => ['$mod' => ['$randomValue', 2]],
+            ],
+            'oddCount' => [
+                '$sum' => ['$subtract' => [1, ['$mod' => ['$randomValue', 2]]]],
+            ],
+            'maxValue' => ['$max' => '$randomValue'],
+            'minValue' => ['$min' => '$randomValue'],
+        ],
+    ],
+];
+
+$cursor = $collection->aggregate($pipeline, ['batchSize' => 2]);
```

Comment:
Is `batchSize` relevant here (also in `bulk.php` and `with_transaction.php`)? This might be copied from the APM example, where I specifically did it to demonstrate `getMore` commands.

Reply:
Ah, yes. I've removed the batchSize option.